sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -37,6 +37,12 @@ object StaticSQLConf {
     .stringConf
     .createWithDefault(Utils.resolveURI("spark-warehouse").toString)
 
+  val CATALOG_DEFAULT_DATABASE = buildStaticConf("spark.sql.catalog.default.database")
+    .doc("The default database for session catalog.")
+    .version("3.2.0")
+    .stringConf
+    .createWithDefault("default")
```

Comment:
Can we test this in `CliSuite`? AFAIK, spark actually never got a chance to create the default database if not exists, which will be done during hive metastore client initialization. If it is configured to `default2` for example, Spark now will get the opportunity to create and there might be 2 default databases then.

Reply:
IMO, the database need exits when not connect to `default`.